### PR TITLE
[cdc] Optimize Duplicate Field Detection in MySQL CDC Schema Building.

### PR DESCRIPTION
**Pull request by zhuangchong**
### Purpose

Optimize Duplicate Field Detection in MySQL CDC Schema Building.

### CODE REVIEW

File: paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/CdcActionCommonUtils.java [hunk 1]
```diff
@@ -140,8 +140,9 @@ public static String columnCaseConvertAndDuplicateCheck(
         if (caseSensitive) {
             return column;
         }
-        checkArgument(existedFields.add(column.toLowerCase()), columnDuplicateErrMsg.apply(column));
-        return column.toLowerCase();
```
Comment (yuzelin) [inline paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/CdcActionCommonUtils.java:144]: Here I think we can keep the filed name case in error message.

Comment (zhuangchong) [reply paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/CdcActionCommonUtils.java:144]: is reserved, the variable used in the error message is `column`, and the returned result variable is `columnLowerCase`

**Review by yuzelin — APPROVED** [review]
+1

**Comment by zhuangchong** [comment]
Thanks. @yuzelin

**Pull request merged**